scripts/process_nf.py

```diff
@@ -127,6 +126,8 @@
 beam_stop_parms=np.array([beam_stop_y_cen,beam_stop_width])
 
 
+max_RAM = 256 #max amount of memory to available in GB
```

Comment:
What is this hardcoded value?  Don't like it, as it is generally bad practice.  Python has utils for finding available memory:
```
import psutil

mem_info = psutil.virtual_memory()

# to get available memory in bytes, do:
# mem_info.available

```